src/ng/directive/ngRepeat.js

```diff
@@ -332,7 +332,7 @@
  *     For example: `item in items | filter:x as results` will store the fragment of the repeated items as `results`, but only after
  *     the items have been processed through the filter.
  *
- *     Please note that `as [variable name] is not an operator but rather a part of ngRepeat
```

Comment:
*     micro-syntax so it can be used only after all filters (and not as operator, inside an expressions